### ISSUE TITLE
Add drag and drop to images

### ISSUE THREAD

**Issue by danrabbit** [elementary/notifications#83]
This would be especially useful if we start sending notifications for screenshots

**Comment by cassidyjames** [comment]
Could you clarify what you mean here? Like, dragging a set notification image out of the notification?

**Comment by danrabbit** [comment]
Yeah exactly. So for example if you take a screenshot and it throws a notification using that image, we could create a DnD source for that image so that you could then drag that into your web browser or something
